core/src/oauth/connection.rs

```diff
@@ -182,6 +202,96 @@ impl Connection {
         &self.metadata
     }
 
+    pub fn access_token_expiry(&self) -> Option<u64> {
+        self.access_token_expiry
+    }
+
+    pub fn access_token_expiry_or_default(&self) -> Option<u64> {
+        self.access_token_expiry
+    }
```

Comment:
Not sure to get the behavior difference from those two methods 🤔

Reply:
noise to be removed